microsoft/PowerToys
# Upgrade check-spelling to v0.0.18

## Summary of the Pull Request

**What is this about:**

Upgrade check-spelling to [v0.0.18](https://github.com/check-spelling/check-spelling/releases/tag/v0.0.18)

**What is include in the PR:** 

* An [advice](https://github.com/check-spelling/spell-check-this/blob/main/.github/actions/spelling/advice.md)
* Note that by default words are only considered if they have at least 3 letters
* Updated excludes based on advice from the bot (any file which has fewer known words than unknown words)
* One typo fix

**How does someone test / validate:** 

1. Create a local fork.
2. Commit a typo.
3. Push a branch with that typo -- observe that it's found

## Quality Checklist

- [x] **Docs:** https://github.com/check-spelling/check-spelling/wiki

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
